README.md

```diff
@@ -1,10 +1,12 @@
-
-## Sealerate
+Sealerate
+=========
```

Comment:
Oh I see it raises it to an "h1" instead of "h2" like the others. Agree with that, but I'm going to switch this to `#` so headings are using consistent notation. I think they're the same?